vscode-extension/package.json

```diff
@@ -33,5 +33,17 @@
     "tslint": "^5.16.0",
     "typescript": "^3.6.4",
     "vscode": "^1.1.35"
+  },
+  "contributes": {
+    "configuration": {
+      "title": "clj-kondo",
+      "properties": {
+        "clj-kondo.config.configArgs": {
+          "type": "string",
+          "scope": "application",
+          "markdownDescription": "Specifies config options to pass to clj-kondo on lint. If specifying a path for an additional edn file, do not include .clj-kondo, that will be included automatically."
```

Comment:
This description isn't really clear on what the input supports: plain EDN, a relative path (resolved against the .clj-kondo dir) and/or absolute paths.

Reply:
I'll clarify along with changes above.